prime-router/src/main/kotlin/history/SubmissionHistory.kt

```diff
@@ -303,17 +314,73 @@ class DetailedSubmissionHistory(
     }
 
     /**
-     * Enrich a parent detailed history with details from the process or translate action.
+     * Enrich a parent detailed history with details from the route and translate actions.
+     * Add destinations, errors, and warnings, to the history details.
+     * Note: Route/Translate is exclusive to the Universal pipeline
+     * See enrichWithProcessAction for the COVID pipeline counterpart
```

Comment:
I'm so used to hearing COVID pipeline that I've been taking that name for granted. Need to check if we have an official name for it...

Reply:
Updating this to read 'TopicReceiver pipeline' instead of 'covid pipeline' would be more accurate, but generally everyone calls it the 'covid pipeline' so I think the message is understood.

Reply:
Updated the name just to get started on the habit